scripts/scil_tractogram_dps_math.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Add, extract or delete information to each streamline from a tractogram
+file. Can be for example SIFT2 weights, processing information, bundle IDs,
+tracking seeds, etc.
+
+Input and output tractograms must always be .trk to simplify filetype checks
+for each operation.
```

Comment:
The real reason is that tck format do not support dps, no? Maybe delete the end of the sentence "to simplify ... "

Reply:
It's because the original script could take .tck as input and add dps to form a .trk. I don't do that because it's too complicated with all the different operations.

I'll make it clearer. :)

Reply:
If it's the same script I don't think we should remove the option for .tck, it could currently be used by someone in some pipeline somewhere and that change could break those pipelines.

Reply:
If the operation is not `add`, a .tck input should always raise an error. However, for saving I think it's ok to always save .trk.

Reply:
@CHrlS98 It would break any pipeline using it either way, it's not the same script and it has different options. But alright I don't think it's a bad idea to leave .tck support for add.

Reply:
Ah oui c'est vrai c'est même plus le même script. Do as you wish, if it's too complicated to support .tck you can let it go.